frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseOverrideForkPointAction.java

```diff
@@ -97,25 +93,21 @@ public void run(ProgressIndicator indicator) {
     }.queue();
   }
 
-  /**
-   * This method must NOT be called on UI thread.
-   */
+  @NotUiThreadSafe
```

Comment:
Nit: `@NotUIThreadSafe` for consistency with `@UIEffect` & Co. (and IIRC 2 letter abbrevs in general are uppercase in Java)